themes/finna2/templates/collection/view.phtml

```diff
@@ -57,76 +57,84 @@
       }
   }
 
+  // LIDO records are displayed like on the record page:
+  $lidoRecord = $this->driver instanceof \Finna\RecordDriver\SolrLido;
```

Comment:
Pitäisikö $lidoRecord muuttaa ei yksilöivään muotoon ja katsoa onko ajuri joku tietyistä?

Esim. tyyliin jotain seuraavaa
$displayAsRecord = $this->driver instanceof \Finna\RecordDriver\SolrLido || ... jos jatkossa tulee muita muotoja joita halutaan erotella.

Reply:
Hyvä idea, tehty.